api/v1alpha1/etcdcluster_types.go

```diff
@@ -164,6 +164,18 @@ type PodSpec struct {
 	// ExtraEnv are the extra environment variables to pass to the etcd container.
 	// +optional
 	ExtraEnv []corev1.EnvVar `json:"extraEnv,omitempty"`
+
+	// LivenessProbe defines liveness probe check for the pod.
```

Comment:
WDYT about adding something like "if empty, default probe will be used"?

Reply:
It makes sense, the only question how do we want to leave here a reference to this default probe in the comment?

Reply:
I think we can mention endpoint we use for specific probe. For example, "By default /livez endpoint will be used" for liveness probe

@Kirill-Garbar do you have any ideas about that?

Reply:
I like idea to use default liveness/readiness probes for etcd